pkg/lib/web/static_asset_resolver.go

```diff
@@ -26,6 +26,12 @@ var StaticAssetResources = map[string]resource.Descriptor{
 	"app-background-image":      AppBackgroundImage,
 	"app-background-image-dark": AppBackgroundImageDark,
 
+	"settings":                 Settings,
```

Comment:
Use embedded base64 image URL for this small image. Remove this registration here.